react/src/components/KeypairResourcePolicySettingModal.tsx

```diff
@@ -322,30 +336,25 @@ const KeypairResourcePolicySettingModal: React.FC<
           <Input disabled={!!keypairResourcePolicy} />
         </Form.Item>
         <Form.Item label={t('resourcePolicy.ResourcePolicy')} required>
-          <Card
-            styles={{
-              body: {
-                paddingBottom: 0,
-              },
-            }}
-          >
+          <Card>
             {_.map(
               _.chunk(_.keys(resourceSlots), 2),
               (resourceSlotKeys, index) => (
-                <Row gutter={[16, 16]} key={index}>
+                <Row gutter={[24, 16]} key={index}>
                   {_.map(resourceSlotKeys, (resourceSlotKey) => (
                     <Col
-                      span={12}
+                      xs={{ flex: '50%' }}
+                      md={{ flex: '33.3%' }}
                       key={resourceSlotKey}
```

Comment:
You can use span instated of setting percent to `flex`
```suggestion
                      xs={{ span: 12 }}
                      md={{ span: 8 }}
                      key={resourceSlotKey}
```